JourneysSquad/JourneysSquadTests/Model/BillValidationErrorTests.swift

```diff
@@ -0,0 +1,81 @@
+import XCTest
+
+final class BillValidationErrorTests: XCTestCase {
+    func testErrorDescriptionPresentsInAPI() {
+        _ = BillValidationError.tooLongPersonName(name: "someLongName").errorDescription
+        _ = BillValidationError.emptyPersonName(name: " ").errorDescription
+        _ = BillValidationError.negativeAmountOfMoney(number: -10).errorDescription
+        _ = BillValidationError.moreThanTwoDecimalPlacesForMoney(amount: 10.111).errorDescription
+        _ = BillValidationError.invalidSumOfBill(sum: 0)
+        _ = BillValidationError.incorrectEstimatedSumOfTheBill.errorDescription
+    }
+
+    func testErrorDescriptionReturnsCorrespondingMessageForTooLongPersonNameError() {
+        let name = "test"
+
+        let expectedErrorDescription = "The length of the person's name exceeds the maximum allowed length: \(name)."
+
+        let actualErrorDescription = BillValidationError.tooLongPersonName(name: name).errorDescription
+
+        XCTAssertEqual(actualErrorDescription, expectedErrorDescription)
```

Comment:
You can avoid blank lines:

```suggestion
        let name = "test"
        let expectedErrorDescription = "The length of the person's name exceeds the maximum allowed length: \(name)."
        let actualErrorDescription = BillValidationError.tooLongPersonName(name: name).errorDescription

        XCTAssertEqual(actualErrorDescription, expectedErrorDescription)
```

Reply:
Fixed